cfgov/v1/db_router.py

```diff
@@ -7,9 +9,17 @@
     'flags',
     'taggit',
     'jobmanager',
-    'data_research'
+    'data_research',
+    'countylimits',
+    'oahapi',
+    'ratechecker',
 ]
 
+# filter out optional apps that are not installed
+for app_name in cfgov_apps:
+    if app_name not in settings.INSTALLED_APPS:
```

Comment:
turns out this was over-eager in deleting things: for example, the 'auth' app was deleted, because in INSTALLED_APPS it's actually django.contrib.auth